### PR TITLE
make file sizes human readable

### PR DESCRIPTION
**Pull request by saurabhsood91**

### CODE REVIEW

File: sandstone/apps/filebrowser/handlers.py [hunk 1]
```diff
@@ -259,11 +257,10 @@ def get(self):
                         # ignore the current file
                         continue
                     owner = getpwuid(stat_object.st_uid).pw_name
-                    file_size = str((float(stat_object.st_size) / 1024)) + " KiB"
                     curr_file['filename'] = i[0]
                     curr_file['filepath'] = i[1]
                     curr_file['owner'] = owner
-                    curr_file['size'] = file_size
+                    curr_file['size'] = self.fs.get_size(i[1])
```
Comment (zebulasampedro) [inline sandstone/apps/filebrowser/handlers.py]: @saurabhsood91 Does this run for each file in a directory, each time a filetree dir is expanded?

Comment (saurabhsood91) [inline sandstone/apps/filebrowser/handlers.py]: yes. I would presume so

**Comment by saurabhsood91** [comment]
@zebulasampedro another look?

File: sandstone/apps/filebrowser/tests/python/test_posixfs.py [hunk 2]
```diff
@@ -218,18 +219,26 @@ def test_get_dir_contents(self):
         fp = os.path.join(self.test_dir,'testfile.txt')
         open(fp,'w').close()
 
-        contents = PosixFS.get_dir_contents(self.test_dir)
+        user = getpass.getuser()
```
Comment (saurabhsood91) [inline sandstone/apps/filebrowser/tests/python/test_posixfs.py]: @zebulasampedro is there a better way to get the `user`?

Comment (zebulasampedro) [inline sandstone/apps/filebrowser/tests/python/test_posixfs.py]: An alternative approach in the [auth decorator](https://github.com/SandstoneHPC/sandstone-ide/blob/6996f3a951ffd66d69af8cfc53f9584377b06e6d/sandstone/lib/decorators.py#L13).

**Pull request merged**